configure.ac

```diff
@@ -860,7 +860,7 @@ AC_ARG_ENABLE([production],
               [AS_HELP_STRING([--enable-production],
                               [Determines how to run the compiler.])])
 
```

Comment:
Don't you need ", [enable_production $enableval]" following "[Determines how to run the compiler.])]"?

Reply:
Nope. The default behavior suffices.

Reply:
If you look at the configure output in the CI for this PR, it's still broken, but that's because setting CC is broken.